### DISCUSSION TITLE
crash home assistant su Raspberry Pi

### DISCUSSION THREAD

**Discussion by IronMel** [indomus/forum#2539] [HUB: Home Assistant]
Ciao a tutti, ho un problema con HA/raspberry per il quale non saprei da dove iniziare a capire le cause, cerco di spiegare la situazione: 

Ho installato sul Pi HA (docker) e poi acquistato un case con ventola per rendere il sistema più robusto in caso di alte temperature. La connessione avviene in wireless, HA ha un ip dedicato per l'accesso e si collega al cloud per alcune funzioni con Nabu. Il problema è che spesso va in crash e non è più accessibile se non previo riavvio. 

Quando smette di funzionare, prima del riavvio, faccio sempre un controllo collegando monitor e tastiera e il sistema è ancora attivo, solo che risulta irraggiungibile se tento un eventuale accesso remoto tramite putty. Lo stesso vale per l'app. Come posso iniziare a capire da cosa dipende il problema? 

Grazie

**Comment by snakuzzo** [comment]
Innanzi tutto, se hai scritto correttamente il comportamento che noti, ti inviterei a correggere il titolo della discussione.
Visto che non riesci nemmeno ad effettuare la connessione in SSH, non credo proprio che il problema sia HA o docker.
Su dove guardare...bella domanda.
Io inizierei da `/var/log/syslog` e ti dico già di non chiedere "cosa devo cercare?", perché potrebbe essere tutto o niente.
Se hai scelto questa strada dovresti armarti di pazienza e cercare in quel file qualche evidenza di qualche problema, magari in corrispondenza dell'orario in cui hai notato il malfunzionamento
